### PR TITLE
Add refs to parse non attribute parts of SDP

### LINKED ISSUE #227: Resolve TODO: ignorable session-level SDP lines
Right now, just c= is listed, but we probably want a whole lot more.
http://rtcweb-wg.github.io/jsep/#rfc.section.5.6.1

### PR DESCRIPTION
**Pull request by fluffy**
Fixes #227

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -2384,14 +2384,21 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               mismatches between "c=" and the candidate lines;
               because JSEP always uses ICE, "ice-mismatch" is not useful in this context.
               </t>
-              <t>TODO</t>
+              <t>The "i=", "u=", "e=", "p=", "t=", "r=", "z=", "k=",
+	      lines are not used by this specification and MAY be ignored. </t>
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Should we say that they are checked for syntax?

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -2384,14 +2384,21 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
           </t>
           <t>The remaining lines are processed as follows:
             <list>
               <t>The "b=" line, if present, MUST be parsed as specified in
                 <xref target="RFC4566"/>, Section 5.8, and the bwtype and
-                bandwidth values stored.</t>
+              bandwidth values stored.</t>
+	      <t> The "v="  MUST have a version of 0.  </t>
+	      
+	      <t> The "o=" MUST be processed to check it is consistent
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: What is this reference? I looked but didn't see anything relevant to o=.

File: draft-ietf-rtcweb-jsep.xml [hunk 3]
```diff
@@ -2384,14 +2384,21 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+	      lines are not used by this specification and MAY be ignored. </t>
             </list>
           </t>
           <t>The remaining lines are processed as follows:
             <list>
               <t>The "b=" line, if present, MUST be parsed as specified in
                 <xref target="RFC4566"/>, Section 5.8, and the bwtype and
-                bandwidth values stored.</t>
+              bandwidth values stored.</t>
+	      <t> The "v="  MUST have a version of 0.  </t>
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Did you mean to say "line"?

Also, these should probably be sorted in the order in which they appear in SDP.

File: draft-ietf-rtcweb-jsep.xml [hunk 4]
```diff
@@ -2489,13 +2500,26 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                 <xref target="RFC5285"/>, Section 5, and their values stored.</t>
               <t>Any "a=rtcp-fb" attributes MUST be parsed as specified in
                 <xref target="RFC4585"/>, Section 4.2., and their values stored.</t>
-              <t>If present, a single "a=rtcp-mux" line MUST be parsed as specified in
+		<t>If present, a single "a=rtcp-mux" line MUST be parsed as specified in
                 <xref target="RFC5761"/>, Section 5.1.1, and its presence or
                 absence flagged and stored.</t>
-              <t>TODO: a=rtcp-rsize, a=rtcp, a=msid, a=candidate, a=end-of-candidates</t>
+		<t>Any  "a=rtcp-rsize" attributes MUST be parsed as specified in
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Can there be multiple? Otherwise, use "If present, ..."

File: draft-ietf-rtcweb-jsep.xml [hunk 5]
```diff
@@ -2489,13 +2500,26 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+                <xref target="RFC5506"/>, Section 5, and their values stored. </t>
+		<t>Any  "a=rtcp" attributes MUST be parsed as specified in
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: "If present, ..."

But the value is ignored though.

File: draft-ietf-rtcweb-jsep.xml [hunk 6]
```diff
@@ -2489,13 +2500,26 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+                <xref target="RFC3605"/>, Section 2.1, and their values stored. </t>
+		<t>Any  "a=msid" attributes MUST be parsed as specified in
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: "If present"

File: draft-ietf-rtcweb-jsep.xml [hunk 7]
```diff
@@ -2489,13 +2500,26 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+                <xref target="I-D.ietf-mmusic-msid"/>, Section 3.2, and their values stored. </t>
+		<t>Any  "a=candidate" attributes MUST be parsed as specified in
+                <xref target="RFC5245"/>, Section 4.3, and their values stored. </t>
+		<t>Any  "a=remote-candidates" attributes MUST be parsed as specified in
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Do we do anything with these?

File: draft-ietf-rtcweb-jsep.xml [hunk 8]
```diff
@@ -2489,13 +2500,26 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+                <xref target="RFC5245"/>, Section 4.3, and their values stored. </t>
+		<t>Any  "a=end-of-candidates" attributes MUST be parsed as specified in
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: If present

**Comment by juberti** [comment]
LG with a handful of comments

File: draft-ietf-rtcweb-jsep.xml (hunk 2 above)
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: This looks like the right section to me.

File: draft-ietf-rtcweb-jsep.xml (hunk 4 above)
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: Looks like one is what's sensible

File: draft-ietf-rtcweb-jsep.xml (hunk 6 above)
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: Justin, happy to change this but not sure precisely what you want

**Comment by ekr** [comment]
LGTM. Justin, one question for you

**Comment by juberti** [comment]
Did some minor tweaking, now LGTM

**Comment by ekr** [comment]
Rebased and merged by hand

**Pull request closed without merging**